tools/Customization/DevHome.Customization/Views/AddRepositoriesView.xaml.cs

```diff
@@ -42,4 +46,27 @@ public void AssignSourceControlProviderButton_Click(object sender, RoutedEventAr
             ViewModel.AssignSourceControlProviderToRepository(menuItem.Text, repoInfo.RepositoryRootPath);
         }
     }
+
+    public void OpenFolderInFileExplorer_Click(object sender, RoutedEventArgs e)
+    {
+        MenuFlyoutItem menuItem = (MenuFlyoutItem)sender;
+        if (menuItem.DataContext is RepositoryInformation repoInfo)
```

Comment:
I think the first line will crash if it's not a MenuFlyoutItem (I know it currently always would be, but...)
```suggestion
        MenuFlyoutItem menuItem = sender as MenuFlyoutItem;
        if (menuItem?.DataContext is RepositoryInformation repoInfo)
```

Reply:
Incorporated suggestion in latest. Thanks!